tests/integration.rs

```diff
@@ -1208,6 +1241,28 @@ mod tests {
         });
     }
 
+    #[cfg_attr(not(feature = "mmio"), test)]
+    fn test_vhost_user_net_default() {
+        test_vhost_user_net(None, 2, &prepare_vhost_user_net_daemon)
+    }
+
+    #[cfg_attr(not(feature = "mmio"), test)]
+    fn test_vhost_user_net_tap() {
+        let vunet_tap0 = "vunet-tap0";
+        test_vhost_user_net(Some(vunet_tap0), 2, &prepare_vhost_user_net_daemon)
```

Comment:
big nit: Why the local `vunet_tap0` local? It's not even the same name in the two tests that use it so you might as well just do `Some("vunet_tap0")`

Reply:
You mean `Some("vunet-tap0")`, right?